File: mywebroot/CS471/NOTES/EH/CODE/scope2.c
#include <stdio.h>	
int z = 5;
void scope (void) {
	printf("\nz =%d",z);
	int u = z;
	printf("\nz =%d",z);
	float z = 5.5f;
	printf("\nz =%f",z);
	printf("\nu =%d\n \n",u);
	/* can reuse automatic variables */
	{ int z = u;
	  u = 3*z;}
}

	
int  main() {
     scope();
     return 1;

}

File: mywebroot/CS471/NOTES/EH/CODE/Scope2.java
public class Scope2 {
static int z = 5;

	public static void main(String[] args) {
		System.out.println(z);
		int u = z;
		System.out.println(z);
		float z = 5.5f;
		System.out.println(z);
		System.out.println(u);
		/* ERROR can not reuse automatic variable names */
		{ int z = u;}
	}
}
